### ISSUE TITLE
Train and validation set are imbalanced

### ISSUE THREAD

**Issue by zhutmost** [zhutmost/lsq-net#5]
When using the `random_split` function provided in PyTorch to split the dataset to training and validation sets, the propotion of categories are imbalanced.

**Comment by zhutmost** [comment]
Fixed in commit https://github.com/zhutmost/lsq-net/commit/02f802e6652d3d067b953f756f01dd2d50d54456.

**Comment by zhutmost** [comment]
However, the imbalance among categories still exists. Maybe need another `Sampler`.
